features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/metric/interactiontonextview/TimeBasedInteractionIdentifierTest.kt

```diff
@@ -132,7 +132,7 @@ internal class TimeBasedInteractionIdentifierTest : ObjectTest<TimeBasedInteract
 
     @Test
     fun `M return false W defaultThresholdUsed{custom delay used}`(
-        @LongForgery(min = 0, max = 100) timeThresholdMs: Long
+        @LongForgery(min = 1, max = 100) timeThresholdMs: Long
```

Comment:
seems like this change is not related to the PR? is it on purpose?

Reply:
yep, this is small side change, that prevent this test to be flacky ( if min = 0 test gonna fail)